cpp/tests/CMakeLists.txt

```diff
@@ -38,28 +47,97 @@ add_library(cugraphtestutil STATIC
             ../../thirdparty/mmio/mmio.c)
 
 target_compile_options(cugraphtestutil
-            PUBLIC "$<$<COMPILE_LANGUAGE:CXX>:${CUGRAPH_CXX_FLAGS}>"
-                   "$<BUILD_INTERFACE:$<$<COMPILE_LANGUAGE:CUDA>:${CUGRAPH_CUDA_FLAGS}>>"
+    PUBLIC "$<$<COMPILE_LANGUAGE:CXX>:${CUGRAPH_CXX_FLAGS}>"
+           "$<$<COMPILE_LANGUAGE:CUDA>:${CUGRAPH_CUDA_FLAGS}>"
```

Comment:
Also confused as to how this was working for installs before... I guess nobody was building against the test util?

Reply:
That is my assumption as well